addons/web/static/tests/core/ui_service_tests.js

```diff
@@ -267,3 +267,52 @@ QUnit.test("UI active element: trap focus - no focus element", async (assert) =>
     await nextTick();
     assert.strictEqual(document.activeElement, target.querySelector("div[id=idActiveElement]"));
 });
+
+QUnit.test("UI active element: trap focus - first or last tabable changes", async (assert) => {
+    let showState;
+    class MyComponent extends Component {
+        setup() {
+            this.show = useState({ a: true, c: false });
+            showState = this.show;
```

Comment:
nitpicking: not neccessary I think ? We can get the instance of `MyComponent` with the `mount` function